src/main/java/pl/touk/parking/domain/Money.java

```diff
@@ -0,0 +1,51 @@
+package pl.touk.parking.domain;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+
+public class Money {
+
+    private final BigDecimal amount;
+
+    private Money(BigDecimal amount) {
+        this.amount = amount;
+    }
+
+    public static Money of(BigDecimal amount) {
+        return new Money(amount.setScale(2, RoundingMode.UP));
+    }
+
+    public static Money zero() {
+        return Money.of(BigDecimal.ZERO);
```

Comment:
a czemu nie zrobić z tego stałej zamiast metody?

Reply:
Faktycznie, to mogłaby być stała.